public/javascripts/main.js

```diff
@@ -0,0 +1,127 @@
+
+function myFunction() {
+  var x = document.getElementById("myLinks");
+  if (x.style.display === "block") {
+    x.style.display = "none";
+  } else {
+    x.style.display = "block";
+  }
+}
+
+var slideIndex = 1;
```

Comment:
What is the purpose of defining variable here. We can simply pass 1 in below function call.

Reply:
I am using this variable multiple times for implementing slider functionality, in case of using hard coded value it would not be possible to achieve the same.